src/contracts/libraries/GPv2Order.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.7.6;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/// @title Gnosis Protocol v2 Order Library
+/// @author Gnosis Developers
+library GPv2Order {
```

Comment:
The name is a bit weird, since orders are defined somewhere else. I'd expect something like `GPv2Uid` since there is nothing really involving orders. Do you plan to use it later to include order-related functions?

Reply:
Yes, I plan on moving `GPv2Encoding.Order` to `GPv2Order.Data`.